src/components/BottomBar/GeoPositionPanel.jsx

```diff
@@ -0,0 +1,310 @@
+import Button from '../common/Input/Button/Button';
+import { Icon } from '@iconify/react';
+import Popover from '../common/Popover/Popover';
+import styles from './GeoPositionPanel.scss';
+import Picker from './Picker';
+import Input from '../common/Input/Input/Input';
+import {FilterList, FilterListData } from '../common/FilterList/FilterList'
+import {
+  setPopoverVisibility,
+  reloadPropertyTree
+} from '../../api/Actions';
+import CenteredLabel from '../common/CenteredLabel/CenteredLabel';
+import Dropdown from '../common/DropDown/Dropdown';
+import { useLocalStorageState } from '../../utils/customHooks';
+import AnimatedCheckmark from '../common/AnimatedCheckmark/AnimatedCheckmark';
+import InfoBox from '../common/InfoBox/InfoBox';
+
+function MultiStateToggle({labels, checked, setChecked, infoText}) {
+  return (
+    <div className={styles.wrapper}>
+      <p className={`${styles.toggleTitle} ${styles.resultsTitle}`} id={'multiStateToggle'}>Mode</p>
+      <InfoBox inpanel panelscroll={'multiStateToggle'} text={infoText} style={{ paddingTop: '3px', paddingRight: '3px' }} />
+      <div className={styles.toggles}>
+      {labels.map(label => 
+        <React.Fragment key ={`${label}fragment`}>
+          <input
+            id={label}
+            key={label}
+            className={styles.toggle_option}
+            name="state-d"
+            type="radio"
+            onChange={() =>
+              setChecked(label)
+            }
+            checked={label === checked ? "checked" : ""}
+          />
+          <label htmlFor={label} key={`${label}label`} onClick={() => setChecked(label)}>{label}</label>
+        </React.Fragment>)}
+        <div className={styles.toggle_option_slider}>
+        </div>
+      </div>
+    </div>);
+}
+
+const Interaction = {
+  flyTo: "Fly To",
+  jumpTo: "Jump To",
+  addFocus: "Add Focus"
+};
+
+function createSceneGraphNodeTable(globe, label, lat, long, alt) {
+  const position = {
+      Identifier: label,
+      Parent: globe,
+      Transform: {
+        Translation: {
+          Type: "GlobeTranslation",
+          Globe: globe,
+          Latitude: lat,
+          Longitude: long,
+          Altitude: 0
+        }
+      },
+      InteractionSphere: alt,
+      GUI: {
+        Path: "/GeoLocation"
+      }
+  };
+  return position;
+}
+
+function Place({address, onClick, found}) {
+  return (
+    <Button 
+      onClick={onClick}
+      className={styles.place}
+    >
+      <div className={styles.placeButton}>
+        <p>{address}</p>
+        {found && 
+          <div style={{width : '20px', height : '20px'}}>
+            <AnimatedCheckmark style={{width : '20px', height : '20px'}} color={'transparent'} isAnimated={false} />
+          </div>}
+      </div>
+    </Button>
+  );
+}
+
+function GeoPositionPanel({ refresh, luaApi, popoverVisible, setPopoverVisibilityProp }) {
+  const [inputValue, setInputValue] = useLocalStorageState('inputValue',"");
```

Comment:
Why not use a regular component state variable? Curious

Using hooks for this seems a bit overly complicated to me, but I'm sure you have a motivation :) (I hardly know anything about React Hooks)

Reply:
Local storage stores the value in the browser so that it is stored over reloads of the GUI. If we reload the GUI it might be nice to have the same input value